controller/bookController_test.go

```diff
@@ -0,0 +1,68 @@
+package controller
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestGetBook(t *testing.T) {
```

Comment:
Are you mocking your services? Remember that in unit testing you should be mocking any other services you use to avoid actually calling external APIs and services.

You should only worry about what this specific function is doing, not the others